src/config/license.rs

```diff
@@ -110,7 +110,7 @@ impl TemplateParser {
     /// "
     /// );
     /// ```
-    pub fn parse(template: &str) -> Result<String, LicenseError> {
+    pub(crate) fn parse(template: &str) -> Result<String, LicenseError> {
```

Comment:
General question: do you know if it compiles without `(crate)`, given that `TemplateParser` is already public for the crate only ?

Reply:
The compiler complains if `parse` is `pub`:
```
warning: unreachable `pub` item
   --> src/config/license.rs:113:5
    |
113 |     pub fn parse(template: &str) -> Result<String, LicenseError> {
    |     ---^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |     |
    |     help: consider restricting its visibility: `pub(crate)`
    |
```

Reply:
nice thanks, love this.